Documentation/Manual/Texinfo/chap-inspector.texi

```diff
@@ -4,11 +4,17 @@
 @cindex Clouseau
 
 The inspector, called ``Clouseau'', is used for interactively inspecting
-objects. It lets you look inside objects, inspect slots, disassemble and
-trace functions, view keys and values in hash tables, and quite a few
-other things as well. It can be extended to aid in debugging of specific
-programs, similar to the way the Lisp printer can be extended with
-@genfun{print-object}.
+Common Lisp objects.  It lets you look inside objects, inspect slots,
+disassemble and trace functions, view keys and values in hash tables,
+and quite a few other things as well.  It can be extended to aid in
+debugging of specific programs, similar to the way the Lisp printer can
+be extended with @genfun{print-object}.  The inspector can be used as a
```

Comment:
Minor nitpick, You started with "The inspector, called `Clouseau`". You can replace "The inspector" here and in following paragraph with just "Clouseau".

Reply:
Done.